### ISSUE TITLE
mp4box.exe can not phase Restricted Video Sample Entry Box("resv")

### ISSUE THREAD

**Issue by yyyykf** [gpac/gpac#2489]

**Comment by rbouqueau** [comment]
Could you provide more details about your issue please?

**Comment by yyyykf** [comment]
when I use the command to phase a file encapsulated from a VVC bitstream:
`mp4box.exe -diso -std .\bitstream1.mp4`
I got an error:
```
[VVC] Invalid NALU type 17 in vvcC - ignoring
[iso file] Unknown box type spkt in parent schi
[iso file] Read Box "trak" (start 4084) failed (Invalid IsoMedia File) - skipping
[iso file] Read Box "moov" (start 36) failed (Invalid IsoMedia File) - skipping
Error opening file .\bitstream1.mp4: Invalid IsoMedia File
```
the attachment is the MP4 file I want to phase and the result I phased using _ISOMovies.h_, I think it may be caused by the "resv" box. Could you help me with it? 

https://github.com/gpac/gpac/assets/38534042/ee5ed8eb-9f0c-4152-83a3-9f38dbd9a47c

[bitstream1.txt](https://github.com/gpac/gpac/files/11648002/bitstream1.txt)

**Comment by jeanlf** [comment]
you need latest master, this was fixed in 140ddb5f
